modules/acquia_cms_dam/acquia_cms_dam.module

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * Contains hook implementations for ACMS DAM.
+ */
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_dam_modules_installed($modules) {
+  // Update content type image field configuration for DAM integration.
+  $moduleHandler = \Drupal::service('module_handler');
+  // Update article image field to support Acquia DAM.
+  if ($moduleHandler->moduleExists('acquia_cms_article')) {
+    $fieldConfig = \Drupal::configFactory()->getEditable('field.field.node.article.field_article_image');
+    if ($fieldConfig) {
+      _acquia_cms_dam_update_image_field($fieldConfig);
+    }
+  }
+  // Update event image field to support Acquia DAM.
+  if ($moduleHandler->moduleExists('acquia_cms_event')) {
+    $fieldConfig = \Drupal::configFactory()->getEditable('field.field.node.event.field_event_image');
+    if ($fieldConfig) {
+      _acquia_cms_dam_update_image_field($fieldConfig);
+    }
+  }
+  // Update page image field to support Acquia DAM.
+  if ($moduleHandler->moduleExists('acquia_cms_page')) {
+    $fieldConfig = \Drupal::configFactory()->getEditable('field.field.node.page.field_page_image');
+    if ($fieldConfig) {
+      _acquia_cms_dam_update_image_field($fieldConfig);
+    }
+  }
+  // Update person image field to support Acquia DAM.
+  if ($moduleHandler->moduleExists('acquia_cms_person')) {
+    $fieldConfig = \Drupal::configFactory()->getEditable('field.field.node.person.field_person_image');
+    if ($fieldConfig) {
+      _acquia_cms_dam_update_image_field($fieldConfig);
+    }
+  }
+  // Update place image field to support Acquia DAM.
+  if ($moduleHandler->moduleExists('acquia_cms_place')) {
+    $fieldConfig = \Drupal::configFactory()->getEditable('field.field.node.place.field_place_image');
+    if ($fieldConfig) {
+      _acquia_cms_dam_update_image_field($fieldConfig);
+    }
+  }
+}
+
+/**
+ * Helper function to update content type image field.
+ */
+function _acquia_cms_dam_update_image_field($fieldConfig) {
```

Comment:
@rajeshreeputra Function parameters should be snake_case for example $field_config

Reply:
updated